test/Makefile

```diff
@@ -48,6 +55,16 @@ init-submodules:
 	git submodule update --init $(GTEST_MODULE)
 	touch init-submodules
 
+$(YAML_CPP_LIBS):
+	cd $(SRC_DIR) && $(MAKE) $@
+
+build-yaml-cpp: $(YAML_CPP_DIR)/Makefile
```

Comment:
Since you're delegating `$(YAML_CPP_LIBS)` to `src/Makefile`, the `build-yaml-cpp` and `yaml-cpp` targets are no longer needed.

Reply:
Fixed, thanks.